### ISSUE TITLE
error in IE8.js and IE7.js

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#23]
```
What steps will reproduce the problem?
1. Loading the attached testpage

What is the expected output? What do you see instead?
internet explorer 6.0.2800 produce an error which is marked in the status
line and stops executing other js scripts.

error on line 530 in IE8.js (IE7.js)
the object does not support this method or property


What version of the product are you using? On what operating system?
IE7 version 2.0 (beta) Jan 06

Please provide any additional information below.

when you delete the styles in my example, then you get no error.
```

Original issue reported on code.google.com by `philipp....@gmail.com` on 1 Feb 2008 at 11:49

Attachments:
- [test.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-23/comment-0/test.html)

**Comment by GoogleCodeExporter** [comment]
```
I can't recreate this bug.
```

Original comment by `dean.edw...@gmail.com` on 1 Feb 2008 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
maybe i translated the error not correct and i attach a screenshot of the error
message. i can reproduce the error on two computers with windows 2000 and 
internet
explorer 6.0.2800

the right translation of the error message:
Object doesn't support this property or method


```

Original comment by `philipp....@gmail.com` on 2 Feb 2008 at 8:16

Attachments:
- [bug_ie7.png](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-23/comment-2/bug_ie7.png)

**Comment by GoogleCodeExporter** [comment]
```
I don't want a screenshot of an error message. I want a test case. I don't get 
any
error with the sample HTML/CSS you originally provided.
```

Original comment by `dean.edw...@gmail.com` on 2 Feb 2008 at 8:36
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
i'm having this exact same javascript message, at : 
http://test.rstin.com 
with IE7 7.0.5730.13
i have tried without loading any other javascript files and i get the same 
javascript error

```

Original comment by `raphaele...@gmail.com` on 10 Feb 2008 at 4:47

**Comment by GoogleCodeExporter** [comment]
```
it might be because i use hacks in some of my css style... so i don't think it 
is 
even a bug of IE7/IE8.
```

Original comment by `raphaele...@gmail.com` on 10 Feb 2008 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in the trunk:

http://ie7-js.googlecode.com/svn/trunk/lib/IE8.js
```

Original comment by `dean.edw...@gmail.com` on 10 Feb 2008 at 11:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
it was fixed for me in IE7 version 2.0 beta3
```

Original comment by `philipp....@gmail.com` on 11 Feb 2008 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
yes it has !!! thank you for being so quick !!
```

Original comment by `raphaele...@gmail.com` on 18 Feb 2008 at 7:11
